java/ddlogapi/DDlogAPI.java

```diff
@@ -129,6 +126,7 @@ private void checkHandle() throws DDlogException {
      */
     public DDlogAPI(int workers, Consumer<DDlogCommand<DDlogRecord>> callback, boolean storeData)
             throws DDlogException {
+        System.loadLibrary("ddlogapi");
```

Comment:
Will this work if DDlogAPI is instantiated multiple times with the same version of the library?

Reply:
I don't really know. Do we have such a test case yet?

Reply:
we do

Reply:
and I thought you also had a test like that at some point